bom-latest/pom.xml

```diff
@@ -85,6 +87,17 @@
                 <classifier>tests</classifier>
                 <version>${workflow-job-plugin.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins.workflow</groupId>
+                <artifactId>workflow-multibranch</artifactId>
+                <version>${workflow-multibranch-plugin.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins.workflow</groupId>
+                <artifactId>workflow-multibranch</artifactId>
+                <classifier>tests</classifier>
+                <version>${workflow-multibranch-plugin.version}</version>
+            </dependency>
```

Comment:
Not correctly running PCT. [Failed](https://ci.jenkins.io/job/Tools/job/bom/job/PR-179/33/execution/node/10118/log/) yet this did not get recorded as a test failure. Something I need to look into…